types/qunit/index.d.ts

```diff
@@ -1,7 +1,8 @@
-// Type definitions for QUnit v2.5.0
+// Type definitions for QUnit v2.9.2
```

Comment:
QUnit.module.only is available since 2.4.0: https://github.com/qunitjs/qunit/releases/tag/2.4.0

Reply:
ok, in the future, it would be most appropriate to change the version in the `.d.ts` header to `2.4.0` (although we can't go back now that this is published)

Reply:
Ok, I’m still a bit confused about the real meaning of this number then. Is there a definition what to put there somewhere?

Reply:
> Ok, I’m still a bit confused about the real meaning of this number then.

It's meant to describe the version of the library that the types align with, and it also bumps the version of the `@types/qunit` package. 

In this case, it means we skipped straight from the 2.5.x releases straight to 2.9.0

![Screen Shot 2019-05-10 at 11 00 54 AM](https://user-images.githubusercontent.com/558005/57547258-f5165d80-7312-11e9-8323-b1921560115f.png)

Despite the fact that this API is available in earlier versions of QUnit, we've sent a signal indicating that QUnit 2.9 is the release that _introduces_ this `QUnit.module.only` feature, and that users of older QUnit versions should stick with the corresponding versions of the types.

More info here: https://github.com/DefinitelyTyped/DefinitelyTyped/#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library

Reply:
Ok. I suggest that I add the features that are still missing to be valid typings for `2.9.0` and submit a new PR for that. There are a few of them still.